### PR TITLE
[hi] Localize /en/docs/reference/glossary/statefulset.md

### PR DESCRIPTION
**Pull request by surbhiahuja**

### CODE REVIEW

File: content/hi/docs/reference/glossary/statefulset.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- मौलिक(fundamental)
+- कोर-ऑब्जेक्ट(core-object)
+- काम का बोझ(workload)
+- भंडारण(storage)
+---
+{{<glossary_tooltip text="Pods" term_id="pod" >}} के सेट के परिनियोजन और स्केलिंग का प्रबंधन करता है, *और इन पॉड्स के क्रम और विशिष्टता* के बारे में गारंटी प्रदान करता है।
+
+<!--more--> 
+
+एक {{<glossary_tooltip term_id="तैनाती">}} की तरह, एक StatefulSet एक समान कंटेनर विनिर्देश पर आधारित पॉड्स का प्रबंधन करता है। परिनियोजन के विपरीत, स्टेटफुलसेट अपने प्रत्येक पॉड के लिए एक चिपचिपा पहचान रखता है। ये पॉड एक ही विनिर्देश से बनाए गए हैं, लेकिन विनिमेय नहीं हैं&#58; प्रत्येक का एक स्थायी पहचानकर्ता होता है जिसे वह किसी भी पुनर्निर्धारण के दौरान बनाए रखता है।
```
Comment (bishal7679) [inline content/hi/docs/reference/glossary/statefulset.md:20]:
```suggestion
एक {{<glossary_tooltip text="डिप्लॉयमेंट" term_id="deployment">}} की तरह, एक स्टेटफुलसेट एक सदृश कंटेनर विनिर्देश पर आधारित पॉड्स का प्रबंधन करता है। डिप्लॉयमेंट के विपरीत, स्टेटफुलसेट अपने प्रत्येक पॉड के लिए एक चिपचिपा पहचान बनाए रखता है। ये पॉड एक ही विनिर्देश से बनाए गए हैं, लेकिन विनिमय करने योग्य नहीं हैं&#58; प्रत्येक का एक स्थायी पहचानकर्ता होता है जिसे वह किसी भी पुनर्निर्धारण के दौरान बनाए रखता है।
```

File: content/hi/docs/reference/glossary/statefulset.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+यदि आप अपने कार्यभार के लिए दृढ़ता प्रदान करने के लिए स्टोरेज वॉल्यूम का उपयोग करना चाहते हैं, तो आप समाधान के हिस्से के रूप में स्टेटफुलसेट का उपयोग कर सकते हैं। हालांकि स्टेटफुलसेट में अलग-अलग पॉड विफलता के लिए अतिसंवेदनशील होते हैं, लगातार पॉड पहचानकर्ता मौजूदा वॉल्यूम को नए पॉड्स से मिलाना आसान बनाते हैं जो किसी भी विफल हो गए हैं।
```
Comment (bishal7679) [inline content/hi/docs/reference/glossary/statefulset.md:22]:
```suggestion
यदि आप अपने वर्कलोड को दृढ़ता प्रदान करने के लिए स्टोरेज वॉल्यूम का उपयोग करना चाहते हैं, तो आप समाधान के हिस्से के रूप में स्टेटफुलसेट का उपयोग कर सकते हैं। हालांकि स्टेटफुलसेट में अलग-अलग पॉड विफलता के लिए अतिसंवेदनशील होते हैं, दृढ़ पॉड पहचानकर्ता मौजूदा वॉल्यूम को नए पॉड्स से मिलाना आसान बनाते हैं जो असफल होने वाले किसी भी पॉड को प्रतिस्थापित करता है।
```

**Review by bishal7679 — CHANGES_REQUESTED** [review]
Thank you @surbhiahuja
Please go through the suggested changes and commit them if you agree.
Feel free to start a discussion in case any doubt!

**Comment by bishal7679** [comment]
Fixes:- #35600
Closes:- #37533 
/retitle Localize /en/docs/reference/glossary/statefulset.md

**Comment by bishal7679** [comment]
Fixes:- https://github.com/kubernetes/website/issues/35600
Closes:- https://github.com/kubernetes/website/issues/37533
/retitle [hi] Localize /en/docs/reference/glossary/statefulset.md

**Comment by surbhiahuja** [comment]
@bishal7679 , I have updated file as suggested - https://github.com/kubernetes/website/pull/37552/commits

File: content/hi/docs/reference/glossary/statefulset.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+title: स्टेटफुलसेट(StatefulSet)
+id: स्टेटफुलसेट(StatefulSet)
```
Comment (sftim) [inline content/hi/docs/reference/glossary/statefulset.md:3]:
```suggestion
id: statefulset
```

**Comment by surbhiahuja** [comment]
@sftim , Updated as suggested.

File: content/hi/docs/reference/glossary/statefulset.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+title: StatefulSet
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/statefulset.md:2]: Please translate this.

File: content/hi/docs/reference/glossary/statefulset.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+- fundamental
+- core-object
+- workload
+- storage
+---
+{{<glossary_tooltip text="पॉड्स" term_id="pod" >}} के एक सेट की डिप्लॉयमेंट और स्केलिंग का प्रबंधन करता है, और इन पॉड्स के *क्रम और विशिष्टता के बारे में गारंटी प्रदान करता है*।
+
+<!--more--> 
+
+एक {{<glossary_tooltip text="डिप्लॉयमेंट" term_id="deployment">}} की तरह, एक स्टेटफुलसेट एक सदृश कंटेनर विनिर्देश पर आधारित पॉड्स का प्रबंधन करता है। डिप्लॉयमेंट के विपरीत, स्टेटफुलसेट अपने प्रत्येक पॉड के लिए एक चिपचिपा पहचान बनाए रखता है। ये पॉड एक ही विनिर्देश से बनाए गए हैं, लेकिन विनिमय करने योग्य नहीं हैं&#58; प्रत्येक का एक स्थायी पहचानकर्ता होता है जिसे वह किसी भी पुनर्निर्धारण के दौरान बनाए रखता है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/statefulset.md:20]:
```suggestion
एक {{<glossary_tooltip text="डिप्लॉयमेंट" term_id="deployment">}} की तरह, एक स्टेटफुलसेट एक सदृश कंटेनर विनिर्देश पर आधारित पॉड्स का प्रबंधन करता है। डिप्लॉयमेंट के विपरीत, स्टेटफुलसेट अपने प्रत्येक पॉड के लिए एक चिपचिपा पहचान बनाए रखता है। ये पॉड एक ही विनिर्देश से बनाए गए हैं, लेकिन विनिमय करने योग्य नहीं हैं; प्रत्येक का एक स्थायी पहचानकर्ता होता है जिसे वह किसी भी पुनर्निर्धारण के दौरान बनाए रखता है।
```

File: content/hi/docs/reference/glossary/statefulset.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+---
+title: StatefulSet
+id: StatefulSet
+date: 2018-04-12
+full_link: /docs/concepts/workloads/controllers/statefulset/
+short_description: >
+  प्रत्येक पॉड के लिए स्थायी संचयन और दृढ़ पहचानकर्ता के साथ, पॉड्स के एक सेट की डिप्लॉयमेंट और स्केलिंग का प्रबंधन करता है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/statefulset.md:7]: We've mentioned संचयन here and स्टोरेज below. We'll need to stick to one term.

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]

File: content/hi/docs/reference/glossary/statefulset.md [hunk 7]
```diff
@@ -1,10 +1,10 @@
 ---
-title: StatefulSet
+title: StatefulSet(स्टेटफुलसेट)
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/statefulset.md:2]:
```suggestion
title: स्टेटफुलसेट (StatefulSet)
```

**Review by divya-mohan0209 — COMMENTED** [review]
Small nit. Otherwise lgtm!

File: content/hi/docs/reference/glossary/statefulset.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
+---
+title: स्टेटफुलसेट (StatefulSet)
+id: StatefulSet
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/statefulset.md:3]:
```suggestion
id: statefulset
```

**Review by divya-mohan0209 — COMMENTED** [review]
ID is capitalized wrong.

**Comment by surbhiahuja** [comment]
id is corrected @divya-mohan0209 . Thanks for inputs.

**Review by bishal7679 — APPROVED** [review]
Now changes LGTM!

**Review by divya-mohan0209 — COMMENTED** [review]
/approve

**Pull request merged**